seatunnel-apis/seatunnel-api-spark/src/main/java/org/apache/seatunnel/spark/batch/SparkBatchSink.java

```diff
@@ -14,12 +14,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.seatunnel.spark.batch
 
-import org.apache.seatunnel.spark.BaseSparkSink
+package org.apache.seatunnel.spark.batch;
+
+import org.apache.seatunnel.spark.BaseSparkSink;
+
+import scala.Unit;
 
 /**
  * a SparkBatchSink plugin will write data to other system
  *  using Spark DataSet API.
  */
-trait SparkBatchSink extends BaseSparkSink[Unit] {}
+@SuppressWarnings("PMD.AbstractClassShouldStartWithAbstractNamingRule")
+public abstract class SparkBatchSink extends BaseSparkSink<Unit> {
```

Comment:
better to change `<Unit>` to `<Void>`

Reply:
Because of `org.apache.seatunnel.spark.sink.Kafka` is developed by scala, so the return type of output method is Unit, if I change it to Void it will be compiled error as below:
![image](https://user-images.githubusercontent.com/51053924/152683869-9d5e00cf-19f3-4ffa-907d-3e0424f1cf93.png)
So I did not change it.